500 when deleting a todo

Logs:

```
2021-10-24T20:26:13.012723+00:00 heroku[router]: at=info method=POST path="/todos/26/delete/" host=autodo-app.herokuapp.com request_id=201603a6-5f3e-4044-bdf5-b02c2a064b19 fwd="150.252.242.5" dyno=web.1 connect=0ms service=48ms status=500 bytes=244 protocol=http

2021-10-24T20:26:13.003712+00:00 app[web.1]: [2021-10-24 20:26:13 +0000] [9] [ERROR] Error handling request /todos/26/delete/

2021-10-24T20:26:13.003719+00:00 app[web.1]: Traceback (most recent call last):

2021-10-24T20:26:13.003720+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/gunicorn/workers/sync.py", line 136, in handle

2021-10-24T20:26:13.003721+00:00 app[web.1]:     self.handle_request(listener, req, client, addr)

2021-10-24T20:26:13.003721+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/gunicorn/workers/sync.py", line 179, in handle_request

2021-10-24T20:26:13.003721+00:00 app[web.1]:     respiter = self.wsgi(environ, resp.start_response)

2021-10-24T20:26:13.003723+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/sentry_sdk/integrations/django/__init__.py", line 127, in sentry_patched_wsgi_handler

2021-10-24T20:26:13.003724+00:00 app[web.1]:     return SentryWsgiMiddleware(bound_old_app, use_x_forwarded_for)(

2021-10-24T20:26:13.003724+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/sentry_sdk/integrations/wsgi.py", line 140, in __call__

2021-10-24T20:26:13.003725+00:00 app[web.1]:     reraise(*_capture_exception(hub))

2021-10-24T20:26:13.003725+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/sentry_sdk/_compat.py", line 54, in reraise

2021-10-24T20:26:13.003726+00:00 app[web.1]:     raise value

2021-10-24T20:26:13.003726+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/sentry_sdk/integrations/wsgi.py", line 133, in __call__

2021-10-24T20:26:13.003727+00:00 app[web.1]:     rv = self.app(

2021-10-24T20:26:13.003727+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/core/handlers/wsgi.py", line 133, in __call__

2021-10-24T20:26:13.003727+00:00 app[web.1]:     response = self.get_response(request)

2021-10-24T20:26:13.003728+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/sentry_sdk/integrations/django/__init__.py", line 360, in sentry_patched_get_response

2021-10-24T20:26:13.003728+00:00 app[web.1]:     return old_get_response(self, request)

2021-10-24T20:26:13.003729+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/core/handlers/base.py", line 130, in get_response

2021-10-24T20:26:13.003729+00:00 app[web.1]:     response = self._middleware_chain(request)

2021-10-24T20:26:13.003730+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/core/handlers/exception.py", line 49, in inner

2021-10-24T20:26:13.003730+00:00 app[web.1]:     response = response_for_exception(request, exc)

2021-10-24T20:26:13.003730+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/core/handlers/exception.py", line 114, in response_for_exception

2021-10-24T20:26:13.003731+00:00 app[web.1]:     response = handle_uncaught_exception(request, get_resolver(get_urlconf()), sys.exc_info())

2021-10-24T20:26:13.003731+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner

2021-10-24T20:26:13.003732+00:00 app[web.1]:     response = get_response(request)

2021-10-24T20:26:13.003732+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/sentry_sdk/integrations/django/middleware.py", line 175, in __call__

2021-10-24T20:26:13.003732+00:00 app[web.1]:     return f(*args, **kwargs)

2021-10-24T20:26:13.003733+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/whitenoise/middleware.py", line 59, in __call__

2021-10-24T20:26:13.003733+00:00 app[web.1]:     response = self.get_response(request)

2021-10-24T20:26:13.003734+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/core/handlers/exception.py", line 49, in inner

2021-10-24T20:26:13.003734+00:00 app[web.1]:     response = response_for_exception(request, exc)

2021-10-24T20:26:13.003734+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/core/handlers/exception.py", line 114, in response_for_exception

2021-10-24T20:26:13.003735+00:00 app[web.1]:     response = handle_uncaught_exception(request, get_resolver(get_urlconf()), sys.exc_info())

2021-10-24T20:26:13.003735+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner

2021-10-24T20:26:13.003735+00:00 app[web.1]:     response = get_response(request)

2021-10-24T20:26:13.003736+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/sentry_sdk/integrations/django/middleware.py", line 175, in __call__

2021-10-24T20:26:13.003736+00:00 app[web.1]:     return f(*args, **kwargs)

2021-10-24T20:26:13.003736+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/utils/deprecation.py", line 117, in __call__

2021-10-24T20:26:13.003737+00:00 app[web.1]:     response = response or self.get_response(request)

2021-10-24T20:26:13.003737+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/core/handlers/exception.py", line 49, in inner

2021-10-24T20:26:13.003737+00:00 app[web.1]:     response = response_for_exception(request, exc)

2021-10-24T20:26:13.003737+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/core/handlers/exception.py", line 114, in response_for_exception

2021-10-24T20:26:13.003738+00:00 app[web.1]:     response = handle_uncaught_exception(request, get_resolver(get_urlconf()), sys.exc_info())

2021-10-24T20:26:13.003738+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner

2021-10-24T20:26:13.003738+00:00 app[web.1]:     response = get_response(request)

2021-10-24T20:26:13.003739+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/sentry_sdk/integrations/django/middleware.py", line 175, in __call__

2021-10-24T20:26:13.003739+00:00 app[web.1]:     return f(*args, **kwargs)

2021-10-24T20:26:13.003740+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/utils/deprecation.py", line 117, in __call__

2021-10-24T20:26:13.003740+00:00 app[web.1]:     response = response or self.get_response(request)

2021-10-24T20:26:13.003745+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/core/handlers/exception.py", line 49, in inner

2021-10-24T20:26:13.003745+00:00 app[web.1]:     response = response_for_exception(request, exc)

2021-10-24T20:26:13.003746+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/core/handlers/exception.py", line 114, in response_for_exception

2021-10-24T20:26:13.003746+00:00 app[web.1]:     response = handle_uncaught_exception(request, get_resolver(get_urlconf()), sys.exc_info())

2021-10-24T20:26:13.003747+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner

2021-10-24T20:26:13.003747+00:00 app[web.1]:     response = get_response(request)

2021-10-24T20:26:13.003747+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/sentry_sdk/integrations/django/middleware.py", line 175, in __call__

2021-10-24T20:26:13.003748+00:00 app[web.1]:     return f(*args, **kwargs)

2021-10-24T20:26:13.003748+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/utils/deprecation.py", line 117, in __call__

2021-10-24T20:26:13.003748+00:00 app[web.1]:     response = response or self.get_response(request)

2021-10-24T20:26:13.003748+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/core/handlers/exception.py", line 49, in inner

2021-10-24T20:26:13.003749+00:00 app[web.1]:     response = response_for_exception(request, exc)

2021-10-24T20:26:13.003749+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/core/handlers/exception.py", line 114, in response_for_exception

2021-10-24T20:26:13.003749+00:00 app[web.1]:     response = handle_uncaught_exception(request, get_resolver(get_urlconf()), sys.exc_info())

2021-10-24T20:26:13.003750+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner

2021-10-24T20:26:13.003750+00:00 app[web.1]:     response = get_response(request)

2021-10-24T20:26:13.003750+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/sentry_sdk/integrations/django/middleware.py", line 175, in __call__

2021-10-24T20:26:13.003750+00:00 app[web.1]:     return f(*args, **kwargs)

2021-10-24T20:26:13.003750+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/utils/deprecation.py", line 117, in __call__

2021-10-24T20:26:13.003751+00:00 app[web.1]:     response = response or self.get_response(request)

2021-10-24T20:26:13.003751+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/core/handlers/exception.py", line 49, in inner

2021-10-24T20:26:13.003751+00:00 app[web.1]:     response = response_for_exception(request, exc)

2021-10-24T20:26:13.003751+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/core/handlers/exception.py", line 114, in response_for_exception

2021-10-24T20:26:13.003751+00:00 app[web.1]:     response = handle_uncaught_exception(request, get_resolver(get_urlconf()), sys.exc_info())

2021-10-24T20:26:13.003751+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner

2021-10-24T20:26:13.003752+00:00 app[web.1]:     response = get_response(request)

2021-10-24T20:26:13.003752+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/sentry_sdk/integrations/django/middleware.py", line 175, in __call__

2021-10-24T20:26:13.003752+00:00 app[web.1]:     return f(*args, **kwargs)

2021-10-24T20:26:13.003752+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/utils/deprecation.py", line 117, in __call__

2021-10-24T20:26:13.003752+00:00 app[web.1]:     response = response or self.get_response(request)

2021-10-24T20:26:13.003752+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/core/handlers/exception.py", line 49, in inner

2021-10-24T20:26:13.003753+00:00 app[web.1]:     response = response_for_exception(request, exc)

2021-10-24T20:26:13.003753+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/core/handlers/exception.py", line 114, in response_for_exception

2021-10-24T20:26:13.003753+00:00 app[web.1]:     response = handle_uncaught_exception(request, get_resolver(get_urlconf()), sys.exc_info())

2021-10-24T20:26:13.003753+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner

2021-10-24T20:26:13.003753+00:00 app[web.1]:     response = get_response(request)

2021-10-24T20:26:13.003754+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/sentry_sdk/integrations/django/middleware.py", line 175, in __call__

2021-10-24T20:26:13.003754+00:00 app[web.1]:     return f(*args, **kwargs)

2021-10-24T20:26:13.003754+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/utils/deprecation.py", line 117, in __call__

2021-10-24T20:26:13.003755+00:00 app[web.1]:     response = response or self.get_response(request)

2021-10-24T20:26:13.003755+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/core/handlers/exception.py", line 49, in inner

2021-10-24T20:26:13.003756+00:00 app[web.1]:     response = response_for_exception(request, exc)

2021-10-24T20:26:13.003756+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/core/handlers/exception.py", line 114, in response_for_exception

2021-10-24T20:26:13.003756+00:00 app[web.1]:     response = handle_uncaught_exception(request, get_resolver(get_urlconf()), sys.exc_info())

2021-10-24T20:26:13.003757+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner

2021-10-24T20:26:13.003757+00:00 app[web.1]:     response = get_response(request)

2021-10-24T20:26:13.003757+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/sentry_sdk/integrations/django/middleware.py", line 175, in __call__

2021-10-24T20:26:13.003757+00:00 app[web.1]:     return f(*args, **kwargs)

2021-10-24T20:26:13.003761+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/utils/deprecation.py", line 117, in __call__

2021-10-24T20:26:13.003761+00:00 app[web.1]:     response = response or self.get_response(request)

2021-10-24T20:26:13.003761+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/core/handlers/exception.py", line 49, in inner

2021-10-24T20:26:13.003761+00:00 app[web.1]:     response = response_for_exception(request, exc)

2021-10-24T20:26:13.003762+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/core/handlers/exception.py", line 114, in response_for_exception

2021-10-24T20:26:13.003762+00:00 app[web.1]:     response = handle_uncaught_exception(request, get_resolver(get_urlconf()), sys.exc_info())

2021-10-24T20:26:13.003762+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner

2021-10-24T20:26:13.003763+00:00 app[web.1]:     response = get_response(request)

2021-10-24T20:26:13.003763+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response

2021-10-24T20:26:13.003763+00:00 app[web.1]:     response = wrapped_callback(request, *callback_args, **callback_kwargs)

2021-10-24T20:26:13.003763+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/sentry_sdk/integrations/django/views.py", line 67, in sentry_wrapped_callback

2021-10-24T20:26:13.003764+00:00 app[web.1]:     return callback(request, *args, **kwargs)

2021-10-24T20:26:13.003764+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/views/generic/base.py", line 70, in view

2021-10-24T20:26:13.003764+00:00 app[web.1]:     return self.dispatch(request, *args, **kwargs)

2021-10-24T20:26:13.003764+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/contrib/auth/mixins.py", line 71, in dispatch

2021-10-24T20:26:13.003765+00:00 app[web.1]:     return super().dispatch(request, *args, **kwargs)

2021-10-24T20:26:13.003765+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/views/generic/base.py", line 98, in dispatch

2021-10-24T20:26:13.003765+00:00 app[web.1]:     return handler(request, *args, **kwargs)

2021-10-24T20:26:13.003766+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/views/generic/edit.py", line 218, in post

2021-10-24T20:26:13.003766+00:00 app[web.1]:     return self.delete(request, *args, **kwargs)

2021-10-24T20:26:13.003766+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/views/generic/edit.py", line 212, in delete

2021-10-24T20:26:13.003767+00:00 app[web.1]:     success_url = self.get_success_url()

2021-10-24T20:26:13.003767+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/views/generic/edit.py", line 221, in get_success_url

2021-10-24T20:26:13.003767+00:00 app[web.1]:     if self.success_url:

2021-10-24T20:26:13.003767+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/utils/functional.py", line 135, in __wrapper__

2021-10-24T20:26:13.003768+00:00 app[web.1]:     res = func(*self.__args, **self.__kw)

2021-10-24T20:26:13.003768+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/urls/base.py", line 86, in reverse

2021-10-24T20:26:13.003768+00:00 app[web.1]:     return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)

2021-10-24T20:26:13.003769+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.8/site-packages/django/urls/resolvers.py", line 694, in _reverse_with_prefix

2021-10-24T20:26:13.003769+00:00 app[web.1]:     raise NoReverseMatch(msg)

2021-10-24T20:26:13.003769+00:00 app[web.1]: django.urls.exceptions.NoReverseMatch: Reverse for 'todos' not found. 'todos' is not a valid view function or pattern name.
```

--- a/autodo/views/todos.py
+++ b/autodo/views/todos.py
@@ -285,4 +285,4 @@ class TodoUpdate(mixins.LoginRequiredMixin, MultiModelFormView):
 
 class TodoDelete(mixins.LoginRequiredMixin, generic.DeleteView):
     model = Todo
-    success_url = reverse_lazy("todos")
+    success_url = reverse_lazy("home")

